src/Badge/Badge.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import { StyledBadge } from './Badge.styles';
+
+export interface Props {
+  background?: string;
+  color?: string;
+  size?: 'standard'; // TODO: fill this out more one design system has caught-up
+}
+
+export const Badge: React.FC<Props> = ({
+  background = 'grayLight',
+  color = 'grayDarker',
+  size = 'standard',
```

Comment:
I am calling these props out so I can give them default values, and then spreading everything else.